demo_mms.py

```diff
@@ -41,7 +41,7 @@ def main():
     builder = sdk.create_multipart_builder()
     builder.set_body({
         'from': {'phoneNumber': mms_number},
-        'to': [{'phoneNumber': phone_number}],
+        'to': [{'phoneNumber': env.RINGCENTRAL_RECEIVER}],
```

Comment:
It should be `env['RINGCENTRAL_RECEIVER']`. Please run the code to confirm which one works

Reply:
Hi @tylerlong  env['RINGCENTRAL_RECEIVER'] is working , will change the code